modules/openedx/application/outputs.tf

```diff
@@ -37,3 +37,7 @@ output "rds" {
 output "redis" {
   value = aws_elasticache_cluster.redis
 }
+
+output "edxapp_course_discovery_s3_access_key" {
```

Comment:
@farhaanbukhsh This output should be on the `s3/outputs.tf` file (they error out here).
There you can add:
```
output "edxapp_course_discovery_s3_access_key" {
  value = aws_iam_access_key.edxapp_course_discovery_s3_access_key
}
```

Reply:
Ahh okay, is there a way I can test such changes?

Reply:
wouldn't this create a issue if `enable_course_discovery` is false and this resource doesn't exist?

Reply:
https://discuss.hashicorp.com/t/is-it-possible-to-display-output-conditionally/2155